versioning/versions/versions.go

```diff
@@ -25,14 +25,18 @@ type Package struct {
 }
 
 type Version struct {
-	Dir            string
-	TemplateSubDir string `yaml:"templateSubDir"`
-	Repo           string
-	Tags           []string
-	From           string
-	TemplateArgs   map[string]string `yaml:"templateArgs"`
-	Packages       map[string]Package
-	ExcludeTests   []string `yaml:"excludeTests"`
+	Dir                  string
+	TemplateSubDir       string `yaml:"templateSubDir"`
+	Repo                 string
+	Tags                 []string
+	From                 string
+	TemplateArgs         map[string]string `yaml:"templateArgs"`
+	Packages             map[string]Package
+	ExcludeTests         []string `yaml:"excludeTests"`
+	Builder              bool
```

Comment:
Instead of using a boolean here, just check the value of the `BuilderImage` instead: its being non-empty should be equivalent to this boolean set to `true`.

Reply:
Fixed